flowetl/flowetl/flowetl/operators/staging/sql/apply_mapping_to_staged_events.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+UPDATE staging_table_{{ds_nodash}}
```

Comment:
I think dispense with this altogether? The `UPDATE` is going to relatively super expensive and a source of errors.

Reply:
Yeah, we can put it back later.